gluten-core/src/main/scala/io/glutenproject/backendsapi/BackendSettingsApi.scala

```diff
@@ -54,7 +55,10 @@ trait BackendSettingsApi {
   // Whether to fallback aggregate at the same time if its child is fallbacked.
   def fallbackAggregateWithChild(): Boolean = false
 
-  def disableVanillaColumnarReaders(): Boolean = false
+  def disableVanillaColumnarReaders(conf: SparkConf): Boolean =
```

Comment:
Can we remove conf arg?
It looks `GlutenConfig.getConf` can be used to get a conf instance inside the method. Thanks!

Reply:
Hm, `disableVanillaColumnarReaders` is used in `DriverPlugin` while initliazing `SparkContext`, the SQLConf is not initlized at that time.

Reply:
Got it. It makes sense.